aspnetcore/toc.md

```diff
@@ -125,6 +125,9 @@
 #### [ASP.NET Web API Help Pages using Swagger](tutorials/web-api-help-pages-using-swagger.md)
```

Comment:
https://github.com/aspnet/Docs/issues/3827
probably me. You should do all TOC work.